packages/manager/apps/telecom/src/app/telecom/pack/move/offers/move-offers.controller.js

```diff
@@ -210,10 +229,52 @@ export default class PackMoveOffersCtrl {
     set(offer, 'displayedPrice', displayedPrice);
   }
 
+  static updateSelectedGtrOption(value, offer, optionName) {
+    angular.forEach(offer.options, (option) => {
+      if (
+        option.name.startsWith('gtr_') &&
+        option.selected !== null &&
+        optionName.startsWith('gtr_')
+      ) {
+        if (option.name !== optionName) {
+          set(option, 'selected', false);
+        }
+      }
+    });
+  }
+
   static isChosen(option) {
     return option.choosedValue > 0;
   }
 
+  static isGtrOption(optionName) {
+    return optionName.startsWith('gtr_') ? 1 : 0;
+  }
+
+  static isGtrOptionExist(offer) {
+    const gtr = (option) => option.name.startsWith('gtr_');
+    return offer.options.some(gtr);
```

Comment:
**suggestion:** Two statements can be combined
```suggestion
    return offer.options.some((option) => option.name.startsWith('gtr_'));
```

Reply:
done